### ISSUE TITLE
gradient overflow

### ISSUE THREAD

**Issue by Hoteryoung** [mmendiet/GFM#7]
Hi, congrats for your great study.
I came into the problem that the gradient sometimes overflowed (see the python snippet below) when I tried to pre-train the GFM mode using GeoTile.
```python
[2024-01-24 14:52:40 simmim_pretrain](main_teacher.py 224): INFO Train: [10/100][700/1148]      eta 0:07:43 lr 0.000049 time 0.2130 (1.0337)    los
s -0.6558 (-0.6616)     grad_norm 0.6889 (0.7676)       mem 11705MB
[2024-01-24 14:54:19 simmim_pretrain](main_teacher.py 224): INFO Train: [10/100][800/1148]      eta 0:05:57 lr 0.000049 time 0.2107 (1.0273)    los
s -0.6804 (-0.6620)     grad_norm 0.5012 (0.7538)       mem 11705MB
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 262144.0
[2024-01-24 14:56:02 simmim_pretrain](main_teacher.py 224): INFO Train: [10/100][900/1148]      eta 0:04:14 lr 0.000049 time 0.2089 (1.0278)    los
s -0.6462 (-0.6619)     grad_norm 0.4569 (inf)  mem 11705MB
[2024-01-24 14:57:42 simmim_pretrain](main_teacher.py 224): INFO Train: [10/100][1000/1148]     eta 0:02:31 lr 0.000049 time 0.2100 (1.0249)    los
s -0.7272 (-0.6625)     grad_norm 0.5404 (inf)  mem 11705MB
[2024-01-24 14:59:23 simmim_pretrain](main_teacher.py 224): INFO Train: [10/100][1100/1148]     eta 0:00:49 lr 0.000049 time 0.2098 (1.0242)    los
s -0.6921 (-0.6626)     grad_norm 0.5038 (inf)  mem 11705MB
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 262144.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 262144.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 262144.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 262144.0
```

I wonder if this situation happened during your pretraining stage.

FYI
I trained on 2 nodes each with 8 GPUS with batch size set to 32 for the limit of GPU storage.
The BASE_LR and MIN_LR are also shrunk 128/32= 4 fold.

best wishes
Hoter Young
